### ISSUE TITLE
TypeScript-Urql generates invalid type for subscription handler with Operation Suffix config

### ISSUE THREAD

**Issue by Pytal** [dotansimha/graphql-code-generator#4140]
**Describe the bug**
An addendum to #3062.

The generated subscription handler types for TypeScipt-Urql do not respect these config options:

``` yaml
  omitOperationSuffix: true
  operationResultSuffix: Data
```

The generated type with the above config is `<OperationName>Subscription`  when the expected type is `<OperationName>Data`.

**To Reproduce**
Add the above config options to your `codegen.yml`

**Comment by dotansimha** [comment]
@Pytal This issue needs reproduction. I tried to reproduce this, but it also depends on the operation name that you are using.

**Comment by Pytal** [comment]
I think line [76](https://github.com/dotansimha/graphql-code-generator/blob/a4bedec554c6c3cc0d209b9a0f57570ddbeb0b41/packages/plugins/typescript/urql/src/visitor.ts#L76) should be:

```
export function use${operationName}<TData = any>(options: Omit<Urql.Use${operationType}Args<${operationVariablesTypes}>, 'query'> = {}, handler?: Urql.SubscriptionHandler<${operationResultType}, TData>) {
```

Replacing `Urql.SubscriptionHandler<${operationName}` with `Urql.SubscriptionHandler<${operationResultType}`

**Comment by dotansimha** [comment]
Fixed in `v1.15.2`
